api/src/test/java/org/openmrs/api/OrderServiceTest.java

```diff
@@ -1414,22 +1414,23 @@ public void retireOrderFrequency_shouldRetireGivenOrderFrequency() throws Except
 	@Test
 	@Verifies(value = "should unretire given order frequency", method = "unretireOrderFrequency(OrderFrequency)")
 	public void unretireOrderFrequency_shouldUnretireGivenOrderFrequency() throws Exception {
-		OrderFrequency orderFrequency = orderService.getOrderFrequency(3);
+		executeDataSet("org/openmrs/api/include/OrderServiceTest-otherOrderFrequencies.xml");
+		OrderFrequency orderFrequency = orderService.getOrderFrequency(103);
 		assertNotNull(orderFrequency);
 		assertTrue(orderFrequency.isRetired());
-		assertEquals("Some Retire Reason", orderFrequency.getRetireReason());
+		assertNull(orderFrequency.getRetireReason());
 		assertNotNull(orderFrequency.getDateRetired());
-		
+
 		orderService.unretireOrderFrequency(orderFrequency);
 		
-		orderFrequency = orderService.getOrderFrequency(3);
+		orderFrequency = orderService.getOrderFrequency(103);
 		assertNotNull(orderFrequency);
 		assertFalse(orderFrequency.isRetired());
 		assertNull(orderFrequency.getRetireReason());
 		assertNull(orderFrequency.getDateRetired());
-		
+
 		//Should not change the number of order frequencies.
-		assertEquals(3, orderService.getOrderFrequencies(true).size());
+    	assertEquals(7, orderService.getOrderFrequencies(true).size());
```

Comment:
The above formatting (TAB) does not look correct. Could you also explain why you changed from 3 to 7?